implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -130,17 +130,17 @@ init({
         Before:<br>
         <code>/search?q=shorts&amp;products.sort.0.field=price&amp;products.sort.0.order=desc</code><br><br>
         After:<br>
-        <code>/search?q=shorts&amp;products.sort.price=desc</code>
+        <code>/search?q=shorts&amp;sort=price~desc</code>
```

Comment:
I would keep products.sort instead of sort, since the products.sort > sort mapping is configured differently